makeflow/src/makeflow.c

```diff
@@ -1599,6 +1626,19 @@ int main(int argc, char *argv[])
 		makeflow_gc_method = MAKEFLOW_GC_ALL;
 	}
 
+	if(wrapper_umbrella && wrapper_umbrella->spec) {
```

Comment:
This is deliberate. So that the code is not broken no matter the umbrella binary is set or not.
If the umbrella binary is set, we will send the binary to the execution node and the execution node will run `./umbrella run ...`;
If the umbrella binary is not set, we will not send the binary to the execution node and the execution node will run `umbrella run ...`.